lib/maxmind/db.rb

```diff
@@ -136,6 +136,24 @@ def initialize(database, options = {})
     # +ip_address+ is a string in the standard notation. It may be IPv4 or
     # IPv6.
     def get(ip_address)
+      record, = get_with_prefix_length(ip_address)
+
+      record
+    end
+
+    # Return the record for the +ip_address+ in the {MaxMind
+    # DB}[http://maxmind.github.io/MaxMind-DB/] and its associated network
+    # prefix length. The record can be one of several types and depends
+    # on the contents of the database.
```

Comment:
What do you think about saying we return an Array containing these two things?

Reply:
Thanks. I amended the commit with these two changes.